subgraph/src/public-lock.ts

```diff
@@ -314,3 +328,59 @@ export function handleLockMetadata(event: LockMetadataEvent): void {
     lock.save()
   }
 }
+
+// Topic0 of ERC20 Transfer event.
+// Transfer(address,address.uint256)
+export const ERC20_TRANSFER_TOPIC0 =
```

Comment:
Hum, I am not a huge fan of using the encoded version here. Any hint on how you found that?

Reply:
I suspect this may not work for all ERC20 tokens.
See https://ethereum.stackexchange.com/questions/110376/why-some-erc20-tokens-have-different-transfer-event-abi-signature-than-transfer